manifest.json

```diff
@@ -40,8 +40,8 @@
                     "en": "Choose a path for Roundcube",
                     "fr": "Choisissez un chemin pour Roundcube"
                 },
-                "example": "/webmail",
```

Comment:
There is also Rainloop.

Reply:
What do you mean ?
Rainloop has /rainloop as default path.

Reply:
I see no reason to give a generic path for one app and not to an other similar app.

Reply:
Well, this is only the default path, and I think `/webmail` is particularly appropriate, as it will fit user's need in 90% of cases.
I would rather set default to `/webmail` for rainloop as well, and I don't see any conflict here, as most users will only install one webmail, and the other 10% obviously know what they are doing and can choose a different path.

Reply:
> the other 10% obviously know what they are doing and can choose a different path.

Personally, I don't like to use optimistic assumption, there is always a danger. I think that it would be more appropriate to do something like `update-alternatives` (on Debian, and else where ?), but of course this would need work in YunoHost and a bit more packaging effort to support a new option.

Reply:
For security, it's better to put webmail than roundcube, no ?  Someone could test roundcube/ and deduce it's a roundcube.

End users don't care of the soft name, i think webmail/ or just mail/ is simpler to remember